parser/parser.y

```diff
@@ -842,12 +842,11 @@ AlterTableSpec:
 			Position:	$4.(*ast.ColumnPosition),
 		}
 	}
-|	"ADD" ColumnKeywordOpt '(' ColumnDef ColumnPosition ')'
```

Comment:
Why remove `ColumnPosition` ?

Reply:
Because we need to be compatible with MySQL. 
```
  | ADD [COLUMN] col_name column_definition
        [FIRST | AFTER col_name]
  | ADD [COLUMN] (col_name column_definition,...)
```